lib/Controls.tsx

```diff
@@ -109,6 +129,18 @@ export const Controls: React.FC<ControlsProps> = ({
             <Screenshot title={labels?.screenshot} />
           </Button>
         )}
+        {api === AXIS_MEDIA_AMP && parameters['videocodec'] === 'h264' ? (
```

Comment:
I would avoid coupling this to a particular api or parameters (which aren't strictly required). Just use `if (volume !== undefined && setVolume !== undefined`.

Reply:
Fixed.